README.md

```diff
@@ -1,20 +1,16 @@
-# Mitiq
+# <a href="https://github.com/unitaryfund/mitiq"><img src="https://github.com/unitaryfund/mitiq/blob/master/docs/source/img/mitiq-logo.png?raw=true" alt="Mitiq logo" width="350"/></a>
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-25-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
```suggestion
[![All Contributors](https://img.shields.io/github/all-contributors/unitaryfund/mitiq)](#contributors-)
```
This will keep it from line breaking on this badge

Reply:
I also think its better to not have a line break, but since this is automatically generated (including the comments) I thought it any changes would just be reverted. Do you know if this is the case?